src/internal/kopia/upload.go

```diff
@@ -91,6 +92,13 @@ func (cp *corsoProgress) FinishedFile(relativePath string, err error) {
 		return
 	}
 
+	atomic.AddInt64(&cp.totalFiles, 1)
```

Comment:
nit: We can also track this counter using the count bus. Please see `PersistedHashedBytes` as an example.

Reply:
and/or: if you put in a TODO(keepers) for me, I can follow up on it.  I'll be introducing a count.bus helper for the intermittent logging, too. but that's stuck in a PR chain atm.

Reply:
I thought about that but referenced this from our counters discussion:
```
However… counting is easy.  There’s really no point to a global map of counts if we only need them in localized instances.
```